R/fullREMIND.R

```diff
@@ -118,9 +116,10 @@ fullREMIND <- function() {
   calcOutput("ShareIndFE",                            round = 3,  file = "p37_shIndFE.cs3r")
   calcOutput("nonEnergyIndFE",                        round = 8,  file = "f37_fedemand_NonEnergyIndst.cs4r")
   calcOutput("Clinker_to_cement_ratio",               round = 2,  file = "p37_clinker-to-cement-ratio.cs3r")
-  # Delete the 'dummy' line
+  # delete the 'dummy' line
   system(paste0('sed -i "/dummy/d" ', getConfig()$outputfolder, "/p37_clinker-to-cement-ratio.cs3r"))
 
+  calcOutput("Capacity", subtype = "capacityByTech",                   round = 6,  file = "p_histCap.cs3r")  # will be deleted after the merge of REMIND-EU
```

Comment:
Was this brought back intentionally?  @jmuessel

Reply:
I think we needed this for pCap variables later. I would leave this in for now. But I noted it to check again in some weeks, when we do sth on pCap variables.

Reply:
why can't you use the file `pm_histCap.cs3r` (see below)

Reply:
I think you are right but I am not sure atm.